index.html

```diff
@@ -1,25 +1,202 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8">
-    <title>Popular Github Users</title>
-
-    <link rel="stylesheet" href="./style.css">
-  </head>
-  <body>
-    <div class="navbar">
-    <nav class="navbar">
-      <h1 class="navbar_title">Popular Github</h1>
-      <h1 class="navbar_people">People <a href= "#"></h1>
-      <h1 class="navbar_repos">Repos <a href= "#"></h1>
 
+<head>
+  <meta charset="utf-8">
+  <title>Popular Github Users</title>
+
+  <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+  <div class="navbar">
+    <nav class="navbar">
+      <h1 class="navbar_title"> Popular Github <a href="#"> </a> </h1>
+      <h1 class="navbar_people"> People <a href= "#"> </a> </h1>
+      <h1 class="navbar_repos"> Repos <a href= "#"> </a> </h1>
     </nav>
   </div>
-  <div class="profile_popularusers">
-    <img src="http://placekitten.com/100/100" alt="Kittens" class="post__author-photo">
-    <div class="profile_user">
-      <h2 class="human_name">Harry Potter</h2>
-      <h3 class="user_name">dracodrools</h3>
-</div>
-  </body>
+
+  <div class="container">
```

Comment:
When you duplicated the profile elements, you accidentally copied the container.

Instead all of the profiles should be in a single container that only sets max width and margin auto.